### PR TITLE
Add dace kernels for cosmoflow

### PR DESCRIPTION
**Pull request by bvanessen**

### CODE REVIEW

File: applications/physics/cosmology/cosmoflow/DaCe_kernels/generate.sh
Comment (pierfied) [inline applications/physics/cosmology/cosmoflow/DaCe_kernels/generate.sh:1]: @bvanessen Were lines 4 & 6 accidentally removed?

File: applications/physics/cosmology/cosmoflow/DaCe_kernels/generate.sh [hunk 1]
```diff
@@ -1,19 +1,27 @@
 #!/bin/sh
 
 # Frontier environment setup
-source $LMOD_PKG/init/sh
```
Comment (pierfied) [inline applications/physics/cosmology/cosmoflow/DaCe_kernels/generate.sh]: @bvanessen Was this line and the next one accidentally removed?

File: applications/physics/cosmology/cosmoflow/DaCe_kernels/generate.sh [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+# 20+12+29=61ms vs. 32? + 65
+#generate_conv3d $B 4 128 128 128 8388608 2097152 16384 128 1 32 4 3 3 3 $B 32 128 128 128 67108864 2097152 16384 128 1 1 1 1 1 1 1 1 1 1 1
+# # 18+10.6+11.5=40 vs. 30
+# generate_conv3d $B 32 64 64 64 8388608 262144 4096 64 1 64 32 3 3 3 $B 64 64 64 64 16777216 262144 4096 64 1 1 1 1 1 1 1 1 1 1 1
+# # # # 4+3.5+1.5=9.2 vs. 10
+# generate_conv3d $B 64 32 32 32 2097152 32768 1024 32 1 128 64 3 3 3 $B 128 32 32 32 4194304 32768 1024 32 1 1 1 1 1 1 1 1 1 1 1
+# # # # 0.9+0.3+1.7=3.0 vs. 3.6
+# generate_conv3d $B 128 16 16 16 524288 4096 256 16 1 256 128 3 3 3 $B 256 16 16 16 1048576 4096 256 16 1 1 1 1 1 1 1 1 1 1 1
+# # # # 0.2+1.038+0.088=1.3 vs. 1.5
+# generate_conv3d $B 256 8 8 8 131072 512 64 8 1 512 256 3 3 3 $B 512 8 8 8 262144 512 64 8 1 1 1 1 1 1 1 1 1 1 1
```
Comment (tbennun) [inline applications/physics/cosmology/cosmoflow/DaCe_kernels/generate.sh:54]:
```suggestion
```

File: applications/physics/cosmology/cosmoflow/DaCe_kernels/generate.sh [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+# 512 MLPerf + distconv + depth-groups=8
+generate_conv3d_fwd $B 4 66 512 512 69206016 17301504 262144 512 1 32 4 3 3 3 $B 32 64 512 512 553648128 17301504 262144 512 1 0 1 1 1 1 1 1 1 1 1 fwd
+generate_conv3d_fwd $B 32 34 256 256 71303168 2228224 65536 256 1 64 32 3 3 3 $B 64 32 256 256 142606336 2228224 65536 256 1 0 1 1 1 1 1 1 1 1 1 fwd
+generate_conv3d_fwd $B 64 18 128 128 18874368 294912 16384 128 1 128 64 3 3 3 $B 128 16 128 128 37748736 294912 16384 128 1 0 1 1 1 1 1 1 1 1 1 fwd
+# generate_conv3d_fwd $B 128 10 64 64 5242880 40960 4096 64 1 256 128 3 3 3 $B 256 8 64 64 10485760 40960 4096 64 1 0 1 1 1 1 1 1 1 1 1 fwd
+# generate_conv3d_fwd $B 256 6 32 32 1572864 6144 1024 32 1 512 256 3 3 3 $B 512 4 32 32 3145728 6144 1024 32 1 0 1 1 1 1 1 1 1 1 1 fwd
+# generate_conv3d_fwd $B 512 4 16 16 524288 1024 256 16 1 512 512 3 3 3 $B 512 2 16 16 524288 1024 256 16 1 0 1 1 1 1 1 1 1 1 1 fwd
+# generate_conv3d_fwd $B 512 3 8 8 98304 192 64 8 1 512 512 3 3 3 $B 512 1 8 8 98304 192 64 8 1 0 1 1 1 1 1 1 1 1 1 fwd
```
Comment (tbennun) [inline applications/physics/cosmology/cosmoflow/DaCe_kernels/generate.sh:63]: If I am not wrong, these can be uncommented

**Review by tbennun — APPROVED** [review]

**Pull request merged**